common/src/main/res/values/strings.xml

```diff
@@ -102,9 +102,14 @@
     <string name="cancel">Cancel</string>
     <string name="changelog">View Full Change Log</string>
     <string name="checking_with_home_assistant">Checking with Home Assistant</string>
+    <string name="choose_entity">Choose entity</string>
     <string name="choose_server">Choose server</string>
     <string name="clear_favorites">Clear Favorites</string>
     <string name="color_temp">Color temperature: %1$d</string>
+    <string name="complication_entity_invalid">Invalid entity</string>
+    <string name="complication_entity_state_content_description">Entity state</string>
+    <string name="complication_entity_state_label">Complication</string>
```

Comment:
I think "Entity state" makes more sense, because this is shown without any context in a list titled 'Complications' when adding a complication (on the Wear OS 3 emulator):
![A list of complication options, with the Home Assistant app option labeled 'Complication' alongside 'Battery' and 'Countdown to date'](https://user-images.githubusercontent.com/8148535/171685293-683de8d2-f04c-4851-820d-762bcf527e20.png)